src/Main.cc

```diff
@@ -183,7 +180,7 @@ void OnMessage(uWS::WebSocket<false, true>* ws, std::string_view sv_message, uWS
                     if (message.ParseFromArray(event_buf, event_length)) {
                         session->OnRegisterViewer(message, head.icd_version, head.request_id);
                     } else {
-                        fmt::print("Bad REGISTER_VIEWER message!\n");
+                        spdlog::error("Bad REGISTER_VIEWER message!");
```

Comment:
I think that these should all be warnings.

Reply:
:+1: